_config.yml

```diff
@@ -9,6 +9,7 @@ kramdown:
 repository: opendocsg/opendoc-theme
 server_PROD: https://search.opendoc.sg
 server_DEV: https://search-staging.opendoc.sg
+offline_search_only: true
```

Comment:
this should be false, as a default. Most of the opendocs should be indexed by ES.

Reply:
Yup!